quanta-t41/config.json

```diff
@@ -35,7 +35,6 @@
     },
     "workflowRoot": "workflows",
     "taskRoot": "tasks",
-    "httpTemplateRoot": "templates",
-    "httpStaticRoot": "static"
```

Comment:
But the skupack insertion will fail. So even there is nothing in the folder, the folder still need be created?